src/test/java/uk/gov/hmcts/reform/laubackend/idam/authorization/AuthServiceTest.java

```diff
@@ -9,62 +9,77 @@
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 import uk.gov.hmcts.reform.authorisation.validators.AuthTokenValidator;
-import uk.gov.hmcts.reform.laubackend.idam.exceptions.InvalidAuthenticationException;
+import uk.gov.hmcts.reform.idam.client.IdamClient;
+import uk.gov.hmcts.reform.idam.client.models.UserInfo;
+import uk.gov.hmcts.reform.laubackend.idam.exceptions.InvalidAuthorizationException;
+import uk.gov.hmcts.reform.laubackend.idam.exceptions.InvalidServiceAuthorizationException;
 
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.fail;
+import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 @ExtendWith(MockitoExtension.class)
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 class AuthServiceTest {
 
     private static final String REFORM_SCAN_BLOB_ROUTER_SERVICE_AUTH = "DFJSDFSDFSDFSDFSDSFS";
```

Comment:
Can you rename REFORM_SCAN_BLOB_ROUTER_XXXX to be something more appropriate i.e. LAU_BACKEND_XXXX for simplification.